tests/empty-file/test.exw

```diff
@@ -0,0 +1,22 @@
+include leap.e
+
+--<do not edit>
+constant canonical_data = {
+{2015,false,"year not divisible by 4 in common year"},
+{1970,false,"year divisible by 2, not divisible by 4 in common year"},
+{1996,true,"year divisible by 4, not divisible by 100 in leap year"},
+{1960,true,"year divisible by 4 and 5 is still a leap year"},
+{2100,false,"year divisible by 100, not divisible by 400 in common year"},
+{1900,false,"year divisible by 100 but not by 3 is still not a leap year"},
+{2000,true,"year divisible by 400 is leap year"},
+{2400,true,"year divisible by 400 but not by 125 is still a leap year"},
+{1800,false,"year divisible by 200, not divisible by 400 in common year"},
+}
+--</do not edit>
+
+set_test_verbosity(TEST_SHOW_ALL)
+for t in canonical_data do
```

Comment:
Whatever "t" means here, we could use the full word.  As tests are exposed to students, they should not be cryptic.  (Probably true of code in general, though.)

Reply:
"test" - point taken, I've put quite a few of those in...